### PR TITLE
Infra discovery: Port scan needs trailing FF/LN

### PR DESCRIPTION
**Pull request by gildub**
https://bugzilla.redhat.com/show_bug.cgi?id=1212947

### CODE REVIEW

**Comment by gildub** [comment]
@miq-bot add_label enhancement

**Comment by gildub** [comment]
@aufi, could you please review or assign?

**Comment by gildub** [comment]
@aufi, could you please review?

**Comment by aufi** [comment]
Looks good to me, merging.

**Comment by gildub** [comment]
@miq-bot add_label gaprindashvili/yes

**Comment by simaishi** [comment]
Gaprindashvili backport details:

```
$ git log -1
$ git log -1
commit 782083892e4344c7456be28b0177c65e2a44b993
Author: Marek Aufart <aufi.cz@gmail.com>
Date:   Thu Feb 1 16:41:46 2018 +0100

    Merge pull request #205 from gildub/discovery_infra_fix_socket
    
    Infra discovery: Port scan needs trailing FF/LN
    (cherry picked from commit 1c1d8fc4f4814cb8900a836ace036f13f5808ce9)
    
    https://bugzilla.redhat.com/show_bug.cgi?id=1552665
```

File: app/models/manageiq/providers/openstack/infra_discovery.rb [hunk 1]
```diff
@@ -3,15 +3,15 @@
 module ManageIQ::Providers::Openstack
   class InfraDiscovery
     def self.probe(ost)
-      res = ""
       if ManageIQ::NetworkDiscovery::PortScanner.portOpen(ost, 6385)
+        res = ""
         Socket.tcp(ost.ipaddr, 6385) do |s|
```
Comment (aufi) [inline app/models/manageiq/providers/openstack/infra_discovery.rb]: Hi @gildub, we found the issue, when SSL is enabled, port used by ironic from public network is 13385 - we should check this port too.


| ironic           | baremetal               | regionOne                                                                     |
|                  |                         |   admin: http://192.168.24.3:6385                                             |
|                  |                         | regionOne                                                                     |
|                  |                         |   internal: http://192.168.24.3:6385                                          |
|                  |                         | regionOne                                                                     |
|                  |                         |   public: https://192.168.24.2:13385

**Comment by gildub** [comment]
@aufi, absolutely, I'll add a PR, thanks

File: app/models/manageiq/providers/openstack/infra_discovery.rb (hunk 1 above)
Comment (aufi) [reply app/models/manageiq/providers/openstack/infra_discovery.rb]: Thanks Gilles!

**Pull request merged**